translate/api/Tests/BatchTranslateWithModelTests.cs

```diff
@@ -0,0 +1,78 @@
+﻿using System;
+using System.IO;
+using System.Collections.Generic;
+using System.Text;
+using Xunit;
+using TranslateV3Samples;
+using GoogleCloudSamples;
+using Google.Cloud.Storage.V1;
+using Google.Apis.Storage.v1.Data;
+using System.Linq;
+
+namespace Tests
+{
+    public class BatchTranslateWithModelTests : IDisposable
+    {
+        private readonly string _projectId = Environment.GetEnvironmentVariable("GOOGLE_PROJECT_ID");
+        private readonly string _modelId = "TRL8772189639420149760";
+        private readonly string _inputUri = "gs://cloud-samples-data/translation/custom_model_text.txt";
+        private readonly string _bucketName;
+        protected Bucket Bucket { get; private set; }
+
+        private readonly CommandLineRunner _sample = new CommandLineRunner()
+        {
+            VoidMain = TranslateV3BatchTranslateTextWithModelMain.Main
+        };
+
+        // Setup
+        public BatchTranslateWithModelTests()
+        {
+            // Create temp bucket
+            using (var storageClient = StorageClient.Create())
+            {
+                _bucketName = "translate-v3-" + TestUtil.RandomName();
+                storageClient.CreateBucket(_projectId, _bucketName);
+            }
+        }
+
+        public void Dispose()
+        {
+            using (var storageClient = StorageClient.Create())
+            {
+                // Clean up output files.
+                var blobList = storageClient.ListObjects(_bucketName, "");
```

Comment:
Remove, this line and the comment above it, and everywhere else where this is happening. I won't signal it anymore.